app/assets/stylesheets/static_pages.scss

```diff
@@ -502,6 +502,10 @@
   margin-bottom: 0;
 }
 
+.alert-none {
```

Comment:
Oops. Just read your comment. Is there a significance for naming the class `alert-none`?